### PR TITLE
Change HSIL Mapping Code

### PR DESCRIPTION
**Pull request by mickohanlon23**
Change mapping for cytology result '_High grade squamous intraepithelial lesion, Carcinoma in situ dysplasia_' to other SNOMED code within HSIL+ value set.

Pilot partner lead indicated that this newer SNOMED code aligns more closely with the meaning of _High grade squamous intraepithelial lesion, Carcinoma in situ dysplasia_. Because both the old SNOMED code and the newer code are within the same HSIL value set, this should not have any real affect on the behavior of our logic.

### CODE REVIEW

File: src/hooks/useCds/translate.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const testCodeResultMapping = [
       },
       {
         text: "High grade squamous intraepithelial lesion, Carcinoma in situ dysplasia",
```
Comment (jhlee-mitre) [inline src/hooks/useCds/translate.js:39]: There are gaps of text for the code 416033009 (reference: https://ncit.cancer.gov/ncitbrowser/pages/concept_details.jsf?dictionary=SNOMED%20Clinical%20Terms%20US%20Edition&version=2022_03_01&ns=SNOMED%20Clinical%20Terms%20US%20Edition&code=416033009)

`Cervicovaginal cytology: High grade squamous intraepithelial lesion or carcinoma`

Or does the text sticks to `High grade squamous intraepithelial lesion, Carcinoma in situ dysplasia` as fixed, but `416033009` is the most closed one?

**Review by jhlee-mitre — COMMENTED** [review]
One minor comment

File: src/hooks/useCds/translate.js (hunk 1 above)
Comment (mickohanlon23) [reply src/hooks/useCds/translate.js:39]: The `text` field here is the actual display text that is coming from the pilot partner, whereas the `code` field indicates the SNOMED CT code we are mapping it to. So the `text` field is not necessarily supposed to represent the display value for that specific SNOMED code.

Comment (jhlee-mitre) [reply src/hooks/useCds/translate.js:39]: Got it.

**Review by jhlee-mitre — APPROVED** [review]
No concerns.

**Pull request merged**